<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-1EPKU6D" failures="0" tests="6" name="com.mitocode.TestNG.Repo.TestEmpresa" time="0.042" errors="6" timestamp="2020-08-06T12:36:23 PET" skipped="0">
  <testcase classname="com.mitocode.TestNG.Repo.TestEmpresa" name="testExistsByIdEmpresa" time="0.032">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.mitocode.TestNG.Repo.TestEmpresa.testExistsByIdEmpresa(TestEmpresa.java:59)
]]>
    </error>
  </testcase> <!-- testExistsByIdEmpresa -->
  <system-out/>
  <testcase classname="com.mitocode.TestNG.Repo.TestEmpresa" name="testSaveEmpresa2" time="0.002">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.mitocode.TestNG.Repo.TestEmpresa.testSaveEmpresa2(TestEmpresa.java:40)
]]>
    </error>
  </testcase> <!-- testSaveEmpresa2 -->
  <system-out/>
  <testcase classname="com.mitocode.TestNG.Repo.TestEmpresa" name="testSaveEmpresa" time="0.001">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.mitocode.TestNG.Repo.TestEmpresa.testSaveEmpresa(TestEmpresa.java:30)
]]>
    </error>
  </testcase> <!-- testSaveEmpresa -->
  <system-out/>
  <testcase classname="com.mitocode.TestNG.Repo.TestEmpresa" name="testFindByRucEmpresa" time="0.001">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.mitocode.TestNG.Repo.TestEmpresa.testFindByRucEmpresa(TestEmpresa.java:72)
]]>
    </error>
  </testcase> <!-- testFindByRucEmpresa -->
  <system-out/>
  <testcase classname="com.mitocode.TestNG.Repo.TestEmpresa" name="testFindAllEmpresa" time="0.004">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.mitocode.TestNG.Repo.TestEmpresa.testFindAllEmpresa(TestEmpresa.java:48)
]]>
    </error>
  </testcase> <!-- testFindAllEmpresa -->
  <system-out/>
  <testcase classname="com.mitocode.TestNG.Repo.TestEmpresa" name="testFindByRazonSocialEmpresa" time="0.002">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.mitocode.TestNG.Repo.TestEmpresa.testFindByRazonSocialEmpresa(TestEmpresa.java:83)
]]>
    </error>
  </testcase> <!-- testFindByRazonSocialEmpresa -->
  <system-out/>
</testsuite> <!-- com.mitocode.TestNG.Repo.TestEmpresa -->
